JavaScript/ds/Hash.js

```diff
@@ -0,0 +1,98 @@
+class HashTable {
```

Comment:
Add a beginner-friendly pitch to help the others know what exactly does your code does